packages-next/admin-ui/src/templates/adminMetaSchemaExtension.ts

```diff
@@ -141,7 +141,12 @@ export function adminMetaSchemaExtension({
         },
       },
       KeystoneMeta: {
-        adminMeta(rootVal: any, args: any, context: any) {
+        adminMeta(
+          rootVal: any,
+          args: any,
+          context: any
+          // context: { isAdminUIBuildProcess: true } | KeystoneContext FIXME: Why doesn't this work?
+        ) {
           if (context.isAdminUIBuildProcess || isAccessAllowed === undefined) {
```

Comment:
Answer to comment above 

> Why doesn't this work?

Is this line, `context.isAdminUIBuildProcess` is an internal extension just for the build phase, I decided not to include that in the types on the first pass so we could discuss it.

Possibly worth just dropping that in the main context type, but it's a weird one and I'm not sure we want it to be public API.

Reply:
OH right I see what's happening here, sorry -- I _think_ the answer to the comment above is that you need an `&` not a `|`

Reply:
Oh!!! I misread `context.isAdminUIBuildProcess || isAccessAllowed === undefined` as `context.isAdminUIBuildProcess && isAccessAllowed === undefined`, ok, that explains everything, incoming fix 👍